### PR TITLE
Fix issue 6 https://github.com/PrestaShop/ps_mbo/issues/6

### PR DESCRIPTION
**Pull request by MrBaiame**

### CODE REVIEW

**Comment by marionf** [comment]
QA approved, we just need a CR before merge

File: views/js/controllers/module/module_card.js [hunk 1]
```diff
@@ -244,28 +244,29 @@ var AdminModuleCard = function () {
                             mainElement = jqElementObj.parents('.' + alteredSelector).first();
                             mainElement.remove();
                         });
-                        BOEvent.emitEvent("Module Uninstalled", "CustomEvent");
+//                        BOEvent.emitEvent("Module Uninstalled", "CustomEvent");
```
Comment (eternoendless) [inline views/js/controllers/module/module_card.js]: Why are these commented out?

Comment (MrBaiame) [reply views/js/controllers/module/module_card.js]: I don't need it on my module, I handle the module count differently

Comment (eternoendless) [reply views/js/controllers/module/module_card.js]: It's best not to leave out commented code if it's not needed anymore

**Comment by eternoendless** [comment]
By the way, this repo needs a dev branch

File: views/js/controllers/module/module_card.js (hunk 1 above)
Comment (MrBaiame) [reply views/js/controllers/module/module_card.js]: Ok i'll remove them

**Comment by MrBaiame** [comment]
It already has one, I can see it, maybe I need to do something else for the branch to be visible by everybody ?

**Pull request merged**